chapters/ch13/1.setting-up-the-outdoors-dataset.ipynb

```diff
@@ -193,17 +93,17 @@
        "  'body': \"If I wanted to learn how to tie certain knots, or learn about new knots and what they're used for, what are some good resources to look up?\",\n",
        "  'title': \"What's a good resource for learning to tie knots for climbing?\",\n",
        "  'post_type': 'question'},\n",
-       " {'id': '24440',\n",
```

Comment:
These ranked documents are shown in the book (Listing 13.2), so the order needs to be preserved. Not sure what changed in the query (should just be a standard bm25 query) or underlying data, but please check.